tokio-trace/src/subscriber.rs

```diff
@@ -1,9 +1,9 @@
-use super::{Event, Span, StaticMeta};
+use super::{Event, Span, Meta};
 use log;
 use std::time::Instant;
 
 pub trait Subscriber {
-    fn observe_event<'event>(&self, event: &'event Event<'event>);
+    fn observe_event<'event, 'meta: 'event>(&self, event: &'event Event<'event, 'meta>);
```

Comment:
I'm a bit curious as to why there's a new lifetime here.

Reply:
@davidbarsky I've added a lifetime to the `Meta` struct (formerly called `StaticMeta`), since it now has to be generic over lifetimes in the case where it's created from a `log::Metadata`, which has its own lifetime. However, the metadata for an `Event` can't just have the same lifetime as the event, because that means that metadata that _is_ `'static` (i.e. metadata that's created by the `event!` macro) would require the `Event` to _also_ be valid for the `'static` lifetime.

Reply:
Thanks for explaining!

Reply:
added to the docs in 34cae66269c6f0c63234fc1c08ee659cb9389d5e